### ISSUE TITLE
[BUG]: Setting a property can disrupt the style

### ISSUE THREAD

**Issue by Enes5519** [Trendyol/baklava#728]
### Issue description

If we set the 'disabled' of the 'bl-button' to 'undefined,' it doesn't work in the hover state of secondary buttons because it [checks](https://github.com/Trendyol/baklava/blob/next/src/components/button/bl-button.css#L164) for 'aria-disabled=false'.

If you're wondering how to encounter this issue in real life, you can pass 'undefined' for 'disabled' in React to the 'BlButton' component; otherwise, setting the prop from JavaScript is sufficient.

### Baklava Version

2.3.0

### Operating system

- OS: [e.g. iOS]: macOS
- Browser [e.g. chrome, safari]: Firefox

- Any other details...

### Priority this issue should have

Low (slightly annoying)

### Please review the checkboxes that are applicable.

- [ ] This issue is directly related to React

**Comment by BilgeBalga** [comment]
Hi! It seems there is no active PR for this issue. I would love to work on it. Can you assign it to me? @Enes5519
